lib/cosmos/api/tlm_api.rb

```diff
@@ -585,26 +564,6 @@ def tlm_process_args(args, function_name, scope: $cosmos_scope, token: $cosmos_t
       return [target_name, packet_name, item_name]
     end
 
-    def tlm_variable_process_args(args, function_name, scope: $cosmos_scope, token: $cosmos_token)
```

Comment:
Not needed when I re-implemented tlm_variable to use the base tlm methods directly